jobs/etl_jobs/external/dms/utils.py

```diff
@@ -61,3 +13,23 @@ def access_secret_data(project_id, secret_id, version_id="latest", default=None)
         return response.payload.data.decode("UTF-8")
     except DefaultCredentialsError:
         return default
+
+
+def mergeDictionary(dict_1, dict_2):
```

Comment:
Ajoute une doc: `Warning: function not symetrical: dict_2 cannot contain any list)